code/datums/gamemode/role/vampire_role.dm

```diff
@@ -102,13 +102,13 @@
 // -- Not sure if this is meant to work like that.
 // I just put what I expect to see in the "The vampires were..."
 /datum/role/vampire/GetScoreboard()
-	. = ..() // Who he was, his objectives...
-	. += "Total blood collected: <b>[blood_total]</b><br/>"
-	for (var/datum/role/thrall/T in faction.members)
-		. += T.GetScoreboard()
-		. += "<br/>"
+	. = "Total blood collected: <b>[blood_total]</b><br/>"
+	. += ..() // Who he was, his objectives...
 
 /datum/role/vampire/ForgeObjectives()
+	if(!SOLO_ANTAG_OBJECTIVES)
```

Comment:
I'm not sure why this is showing up when it's already on the codebase

Reply:
Did you update your local bleeding-edge before making this PR?
Looks like this PR needs a rebase.

Reply:
This is fixed now